extensions/amp-story/1.0/amp-story.js

```diff
@@ -827,6 +827,16 @@ export class AmpStory extends AMP.BaseElement {
           accessService.areFirstAuthorizationsCompleted();
       accessService.onApplyAuthorizations(
           () => this.onAccessApplyAuthorizations_());
+
+      const firstPage = this.element.querySelector('amp-story-page');
+      // First amp-story-page can't be paywall protected.
+      // Throws an error during development, but just ignore the amp-access rule
+      // in production.
+      dev().assert(
```

Comment:
Honestly, I think this should just be `user()`; if it wasn't so complicated to enforce this with the validator, I think we would do that too.